src/tasks/secondLargestNum.js

```diff
@@ -2,7 +2,15 @@ export const secondLargestNum = (arr) => {
   if (arr.length < 2) {
     throw new Error("Array needs at least 2 elements");
   }
-  const arrWithoutLargest = arr.filter((number) => number != Math.max(...arr));
-  const result = Math.max(...arrWithoutLargest);
-  return result;
+
+  let first = Number.NEGATIVE_INFINITY;
+  let second = Number.NEGATIVE_INFINITY;
```

Comment:
What reasons for adding -Infinity? Don't overcomplicate. Declare first and second with empty values.

Reply:
Unfortunately it doesn't work with empty values.

Reply:
Try this one
![image](https://github.com/linhapeter/js-practice-vention/assets/40803254/d6a36ac7-c130-410c-8ea7-2882c3a4a89e)